### PR TITLE
[5.1] Use Illuminate\Support\Arr::x() instead of the alias array_x()

### PR DESCRIPTION
**Pull request by sevastyanovio**
Same as pull request https://github.com/laravel/framework/pull/9273.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Please send a single commit.

**Comment by GrahamCampbell** [comment]
This is NOT the same as the one you linked. That other PR only changed methods commonly used.

File: src/Illuminate/Database/Connectors/ConnectionFactory.php [hunk 1]
```diff
@@ -150,7 +151,7 @@ protected function mergeReadWriteConfig(array $config, array $merge)
      */
     protected function parseConfig(array $config, $name)
     {
-        return array_add(array_add($config, 'prefix', ''), 'name', $name);
+        return Arr::add(Arr::add($config, 'prefix', ''), 'name', $name);
```
Comment (crynobone) [inline src/Illuminate/Database/Connectors/ConnectionFactory.php]: Any reason why this is more useful than say:

```php
$config['prefix'] = '';
$config['name'] = $name;

return $config;
```

**Comment by davex69** [comment]
The old code use array_x() style, but new code use Arr::x() https://github.com/laravel/framework/commit/b9bddf5478788659441ab15716db2d5ab72d2572.

So why need support two same functions? array_x() style is slowest because you need call one more function, that's why PR https://github.com/laravel/framework/pull/9273 make Laravel a bit faster.

**Comment by GrahamCampbell** [comment]
That's not a new code style.

**Comment by GrahamCampbell** [comment]
> is slowest

Yes I know. :)

**Pull request closed without merging**